### DISCUSSION TITLE
How to create migrations from ddd ?

### DISCUSSION THREAD

**Discussion by npldevfr** [lunarstorm/laravel-ddd#68] [Q&A]
I just discovered the library you made; it’s great! But I was wondering why we can’t create a migration from a model creation? Something like ``php artisan ddd:model MyDomain CustomModel -m`` ? Is there a reason behind this? Otherwise, it would be a great addition!

**Comment by jaspertey** [comment]
> I just discovered the library you made; it’s great! But I was wondering why we can’t create a migration from a model creation? Something like `php artisan ddd:model MyDomain CustomModel -m` ? Is there a reason behind this? Otherwise, it would be a great addition!

This is currently in the works! It's going to be part of a bigger update where `ddd:model` extends Laravel's native `make:model` command and thereby inheriting all the standard options of `make:model`, not just `-m`. I happen to be actively working on it and hope to get something out this week.

**Comment by jaspertey** [comment]
Closing this in favour of the official 1.2 release: https://github.com/lunarstorm/laravel-ddd/releases/tag/v1.2.0
